modules/appeals_api/app/models/concerns/appeals_api/model_validations.rb

```diff
@@ -97,7 +97,7 @@ def validate_retrieve_from_date_range
         return if evidence_submission['retrieveFrom'].nil?
 
         evidence_submission['retrieveFrom'].each_with_index do |retrieval_evidence, evidence_index|
-          retrieval_evidence['attributes']['evidenceDates'].each_with_index do |evidence_date, date_index|
+          retrieval_evidence['attributes']['evidenceDates']&.each_with_index do |evidence_date, date_index|
```

Comment:
So I added a ruby safe nav operator here, when I did the original ticket, i did remember that end date wa no longer required and accounted for that below, but I forgot that the entire evidenceDates object itself was no longer required....